meterfs/meterfs.c

```diff
@@ -237,18 +242,24 @@ int meterfs_updateFileInfo(fileheader_t *f, meterfs_ctx_t *ctx)
 	meterfs_eraseFileTable((headerNew == 0) ? 0 : 1, ctx);
 
 	for (i = 0; i < ctx->filecnt; ++i) {
-		ctx->read(ctx->offset + ctx->hcurrAddr + HGRAIN + (i * HGRAIN), &u.t, sizeof(fileheader_t));
-		if (strncmp(f->name, u.t.name, sizeof(f->name)) == 0)
-			ctx->write(ctx->offset + headerNew + HGRAIN + (i * HGRAIN), f, sizeof(fileheader_t));
-		else
-			ctx->write(ctx->offset + headerNew + HGRAIN + (i * HGRAIN), &u.t, sizeof(fileheader_t));
+		if ((err = ctx->read(ctx->offset + ctx->hcurrAddr + HGRAIN + (i * HGRAIN), &u.t, sizeof(fileheader_t))) < 0)
+			return err;
+		if (strncmp(f->name, u.t.name, sizeof(f->name)) == 0) {
+			if ((err = ctx->write(ctx->offset + headerNew + HGRAIN + (i * HGRAIN), f, sizeof(fileheader_t))) < 0)
+				return err;
+		} else {
```

Comment:
Due to our coding convention `else {` should be in the new line

Reply:
done